docs/admin/access_control/service_accounts.mdx

```diff
@@ -0,0 +1,50 @@
+# Service Accounts
+
+Service accounts are specialized user accounts designed for automation, API integrations, and programmatic access to Sourcegraph, as opposed to using access tokens from regular users. Unlike regular user accounts, service accounts don't require an email address or password and cannot access the Sourcegraph UI. They are specifically intended for automated workflows, CI/CD pipelines, scripts, and other non-human access patterns. Service accounts also don't count towards the user limit and won't be part of any billing cycles.
+
+## Creating Service Accounts
+
+Service accounts are created like regular user accounts, but with a few key differences.
+
+- Go to **Site admin** → **Users & auth** → **Users**
+- Click **Create User**
+- Enter a descriptive **Username** (e.g., `ci-automation`, `api-integration`)
+- Check the **Service account** checkbox
+- Click **Create service account**
+
+You'll be presented with some next steps you might want to take, like creating an access token, managing and assigning roles, and managing repository permissions.
+
+- Service accounts are automatically assigned the "Service Account" system role
+- They appear in the user list with "Service account" type designation
+
+## Managing Access Tokens
+
+Service accounts authenticate using access tokens rather than passwords. For detailed information about creating, managing, and using access tokens, see:
+
+- [Creating an access token](/cli/how-tos/creating_an_access_token)
```

Comment:
ugh, why do these docs live under `cli` 😆 is that the only access token documentation we have? (sigh)